packages/eslint-plugin-query/src/index.ts

```diff
@@ -6,10 +6,10 @@ type RuleKey = keyof typeof rules
 
 interface Plugin extends Omit<ESLint.Plugin, 'rules'> {
   rules: Record<RuleKey, RuleModule<any, any, any>>
-  configs: Record<
-    'recommended' | 'flat/recommended',
-    ESLint.ConfigData | Linter.FlatConfig | Array<Linter.FlatConfig>
-  >
+  configs: {
+    recommended: ESLint.ConfigData | Linter.FlatConfig
```

Comment:
Is `recommended` of type `FlatConfig`?

I don't think so

`plugins` field in the legacy config is `string[]` but in flat config it's `Record<string, Plugin>`

Reply:
Good point. Fixed it now. I stupidly copied it from the old types without considering if it was actually correct.